test/nightwatch-tests/freelancer-view.js

```diff
@@ -40,6 +40,18 @@ module.exports = {
       .pause(2000)
       // test empty review not listed
       .assert.hidden('div.review-text')
+      // claim
+      .waitForElementPresent('div#freelancer-claim', 1000)
+      .assert.containsText('div#freelancer-claim-status-name', 'NOT VERIFIED')
+      .click('button#freelancer-claim-toggle')
+      .waitForElementPresent('div#freelancer-claim-form', 5000)
+      .click('button#freelancer-claim-btn')
+      .pause(2000)
+      .assert.containsText('div#freelancer-claim-form-message', 'No file was given')
+      .setValue('input#freelancer-claim-form-files', require('path').resolve(__dirname + '/README.md'))
```

Comment:
To get the project root, you can use the field `projectRoot` exported by `test/config.js`